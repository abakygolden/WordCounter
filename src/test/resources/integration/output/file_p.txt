PADECE 1
PAN 1
PARA 2
PEL 1
PELEAN 1
PELIGRO 3
PELLENTESQUE 2
PEQUE 1
PERDER 1
PERHAPS 1
PERIL 1
PERILOUS 1
PERO 3
PERSEGUIR 1
PHASELLUS 1
PICARD 1
PICO 1
PIECES 1
PIENSA 1
PIERDE 3
POBRE 1
POCO 1
POR 3
PORQUE 3
POSUERE 3
PRECAVIDOS 1
PRESENTE 2
PREVENIDOS 1
PRIMERA 1
PRIMIS 1
PROCEDAN 1
PROCURAN 1
PROCUREN 1
PROIN 1
PROUD 1
PRUDENCIA 1
PUERTA 1
PUES 4
PURUS 2