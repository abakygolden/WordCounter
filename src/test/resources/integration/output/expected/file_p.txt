PAD 4
PAELLA 4
PANCAKES 4
PAPAYA 4
PARMESAN 8
PASSION 4
PASSIONFRUIT 4
PASTA 8
PAWPAW 4
PEPPERS 4
PERSIMMON 8
PESTO 4
PHILLY 4
PHO 4
PIE 12
PINEAPPLE 8
PIZZA 8
PLANTAIN 4
PLUM 4
PO 4
POMEGRANATE 4
POMELO 4
PORK 4
POT 4
PRICKLY 4
PULLED 4